code/modules/antagonists/paradox_clone/paradox_camera.dm

```diff
@@ -0,0 +1,281 @@
+
+	para_mind.memory = the_original.mind.memory
+
+	para_mind.isblessed = the_original.mind.isblessed
+	para_mind.num_blessed = the_original.mind.num_blessed
+
+	if(the_original.mind.assigned_role)
+		para_mind.assigned_role = the_original.mind.assigned_role
+
+	if(the_original.mind.role_alt_title)
+		para_mind.role_alt_title = the_original.mind.role_alt_title
+
+	if(the_original.mind.initial_account)
+		para_mind.initial_account = the_original.mind.initial_account
+
+	if(the_original.mind.learned_recipes)
+		para_mind.learned_recipes = the_original.mind.learned_recipes
+
+	for(var/obj/item/organ/internal/I in para.internal_organs)
+		I.heal_internal_damage(400) // I'm not using rejuvenate cus it can break some things
+
+	if(para.wear_id)
+		var/obj/item/card/id/W = para.wear_id
+		W.owner_ckey = para.key
+
+	if(the_original.mind.assigned_role == "Clown")
+		var/datum/outfit/job/clown/clo = new()
+		clo.post_equip(clone, FALSE)
+		para_clone_datum.handle_clown_mutation(para, granting_datum = TRUE)
+
+	var/main_objective = pickweight(main_objectives)
+
+	var/escape_objective = pickweight(escape_objectives)
+
+	if(main_objective == "Kill")
+		var/datum/objective/paradox_replace/kill = new()
+		make_owner_and_target(kill, para_clone_datum, the_original.mind)
+		para_clone_datum.add_antag_objective(kill)
+		objective_spell = /datum/spell/paradox_spell/click_target/replace
+
+	if(main_objective == "Protect")
+		var/datum/objective/protect/paradox_shield = new()
+		make_owner_and_target(paradox_shield, para_clone_datum, the_original.mind)
+		paradox_shield.explanation_text += " Check your ability 'United Bonds'. It can help you with this objective."
+		para_clone_datum.add_antag_objective(paradox_shield)
+		objective_spell = /datum/spell/paradox_spell/self/united_bonds // yes, I know this power is kinda useless, but I love even numbers and sometimes it can help...
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	if(escape_objective == "Escape")
+		var/datum/objective/escape/esc = new()
+		esc.owner = para_mind
+		para_clone_datum.add_antag_objective(esc)
+
+	if(escape_objective == "Survive")
+		var/datum/objective/survive/minecraft_mode = new()
+		minecraft_mode.owner = para_mind
+		para_clone_datum.add_antag_objective(minecraft_mode)
+
+	para_mind.objective_holder.objective_owner = para_mind
+
+	para_mind.AddSpell(new objective_spell(para_mind.current))
+	para_mind.spell_list += objective_spell
+	para_clone_datum.current_powers += objective_spell
+
+	var/list/messages = list()
+	messages.Add("<span class='userdanger'>You are a Paradox Clone!</span>")
+	messages.Add(para_clone_datum.greet())
+	messages.Add(para_mind.prepare_announce_objectives())
+	messages.Add("<span class='motd'>For more information, check the wiki page: ([GLOB.configuration.url.wiki_url]/index.php/[para_clone_datum.wiki_page_name])</span>")
+
+	to_chat(para, chat_box_red(messages.Join("<br>")))
```

Comment:
We should avoid calling antag datum code outside of the datum like this. The only thing this proc should be doing is calling `para_mind.add_antag_datum(/datum/antagonist/paradox_clone)` somewhere and, then the antag datum should handle the greeting, etc.